### PR TITLE
Activity restructure

### PR DESCRIPTION
**Pull request by vanderpm**
Current version of stable app. This is the one we would like to use this week

### CODE REVIEW

File: app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java [hunk 1]
```diff
@@ -1,13 +1,321 @@
...
+                    e.printStackTrace();
+                }
+            }else{
+                Toast.makeText(getApplicationContext(),"SD card not found", Toast.LENGTH_LONG).show();
+            }
+            Toast.makeText(getApplicationContext(),"message Saved", Toast.LENGTH_LONG).show();
+
+
+                Intent intent = new Intent(this, AutonActivity.class);
+                startActivity(intent);
```
Comment (KyletheWolff) [inline app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java]: This is going to cause multiple activities

File: app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java [hunk 2]
```diff
@@ -1,13 +1,321 @@
...
+
+
+            //Clear data from form fields
+            clearData(view);
+            finish();
```
Comment (KyletheWolff) [inline app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java]: Calling finish is what you want but I thought there was one other thing we needed.

Comment (vanderpm) [reply app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java]: I do not recall what else we need as I lost all my local branches of the code

File: app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java (hunk 1 above)
Comment (vanderpm) [reply app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java]: removed start Activity and implemented on activity result

File: app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java (hunk 2 above)
Comment (vanderpm) [reply app/src/main/java/com/example/vande/scouting2017/TeleopActivity.java]: Is there anything else we need to add to finish this activity?

**Pull request merged**